libraries/engage/app-rating/helpers/index.js

```diff
@@ -0,0 +1,25 @@
+import { OS_ANDROID, OS_IOS } from '../../../common/constants/Device';
```

Comment:
```suggestion
import { OS_ANDROID, OS_IOS } from '@shopgate/pwa-common/constants/Device';
```
relative imports from another package only work locally when in mono repo. Does not work in prod when all comes from npm